R/coord-funs.R

```diff
@@ -304,3 +324,23 @@ coord_pol_centroids <- function(x, lat = NULL, lon = NULL, drop = TRUE) {
 #   coordinates(wc) <- ~long + lat
 #   wc
 # }
+
+#' @export
+#' @rdname coords
+coord_uncertain <- function(x, coorduncertainityLimit = 30000, drop = TRUE, ignore.na = FALSE){
+  if(!("coordinateUncertaintyInMeters"  %in%  names(x))){
```

Comment:
This check should probably ignore case so that `coordinateuncertaintyinmeters` would be used